docs/book/getting-started/zenml-pro/deployments-overview.md

```diff
@@ -0,0 +1,189 @@
+---
+description: Compare ZenML Pro deployment scenarios to find the right fit for your organization.
+icon: code-merge
+layout:
+  title:
+    visible: true
+  description:
+    visible: true
+  tableOfContents:
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: true
+---
+
+# Deployment Scenarios
+
+ZenML Pro offers three flexible deployment options to match your organization's security, compliance, and operational needs. This page helps you understand the differences and choose the right scenario for your use case.
+
+## Quick Comparison
+
+| Deployment Aspect | SaaS | Hybrid SaaS | Self-hosted |
```

Comment:
Shouldn't you add the purpose column here too, just like you do in all other tables ? This being an introductory section, it feels strange to just enumerate all these components without giving a description of what they are / do.